bin/gwdetchar-omega

```diff
@@ -223,9 +221,9 @@ for block in blocks.values():
                 gps, channel, data[channel.name].astype('float64'), fftlength,
                 resample=block.resample, fthresh=args.far_threshold,
                 search=block.search)
-        except Exception:
-            traceback.print_exc()
-            print('    Problem encountered, skipping {}'.format(channel.name))
+        except Exception as exc:
```

Comment:
Please identify the expected error modes here, rather than just `Exception`, since we only want to catch things associated with channels missing from the source data.